### DISCUSSION TITLE
Feature Request: Element locking

### DISCUSSION THREAD

**Discussion by bacsp** [excalidraw/excalidraw#3872] [General]
Hello, 

i'd like to know whether you have any plans to support pinned elements?
So that you can pin an element and it will stay at a fixed position regardless if you try to drag and drop it.
Or further if you select an area with a bunch of elements and drag these elements, the pinned element (within the actual selection area) will not be selected and not moved.

**Comment by dwelle** [comment]
Yes, it's on the backlog https://github.com/excalidraw/excalidraw/issues/1976, open for implementation (spec: https://github.com/excalidraw/excalidraw/issues/1976#issuecomment-844165742).
